tests/test_advection.py

```diff
@@ -122,7 +122,7 @@ def test_advection_periodic_zonal(mode, xdim=100, ydim=100, halosize=3):
     assert(len(fieldset.U.lon) == xdim + 2 * halosize)
 
     pset = ParticleSet(fieldset, pclass=ptype[mode], lon=[0.5], lat=[0.5])
-    pset.execute(AdvectionRK4 + pset.Kernel(periodicBC), endtime=delta(hours=20), dt=delta(seconds=30))
+    pset.execute(AdvectionRK4 + pset.Kernel(periodicBC), endtime=0.65, dt=0.01)
```

Comment:
why change this? And also the other changes in this `tests/test_advection.py` file?

Reply:
The change you highlighted was a consequence of the change `spherical` to `flat` just above.

I needed to do this change for previous implementation of periodic BC in curvi grids.
And I thought there was consequence since a periodic BC in spherical mode should not reconnect the 1 degree with the 0 as it is here but user might do such periodic BC so we have to be able to do it.
But indeed, it works with currenet implementation, so I reversed it as it was before.